.github/workflows/api-ci.yaml

```diff
@@ -78,5 +78,6 @@ jobs:
 
           cd ../harmony-api
 
-          npx prisma migrate dev
-          npm run test
+          # Commenting this part out, we need to discuss whether all of our tests are self-cleaning before running this on production
```

Comment:
Migrate for production should not happen on CI and only on CD with migrate deploy. After reviewing all tests and agreeing that they are all self-cleaning I can add this to CD.